### PR TITLE
Update READMEs

### LINKED ISSUE #301: Add whys to READMEs
## Description
- Add why reasoning to the READMEs

## Required Files
All:
-`README.md`

## User Story
As an engineer it is important for the client to understand why they need this project or what part of the project is required for.

### PR DESCRIPTION
**Pull request by annalisev**
Made use of code blocks and I also added a 'Why' section to each README.

Let me know if I can make anything clearer.

Closes #301 .

### CODE REVIEW

File: pipeline/extract/epic/conftest.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
+'''A conftest file that defines functions used in test_epic.py.'''
```
Comment (shindym) [inline pipeline/extract/epic/conftest.py:1]: Why is this here?

File: dashboard/README.md [hunk 2]
```diff
@@ -1,31 +1,34 @@
 ## Description
 
-This Folder is responsible for creating the dashboard that displays metrics and graphs about our data.
+This folder is responsible for creating the dashboard that displays metrics and graphs about our data.  
+
+### The Why
+The dashboard is important for business users to see graphs and metrics about the latest releases, this will allow them to identify trends in the market and tailor their games or marketing campaigns to suit these trends.
```
Comment (hhm970) [inline dashboard/README.md:6]: Nice `why` here

File: pipeline/notification/README.md [hunk 3]
```diff
@@ -1,20 +1,22 @@
 ## Description
 
-This Folder is responsible for sending daily email notifications to our users.
+This folder is responsible for sending daily email notifications to our users.
 
+### The Why
+This folder is tailored towards gamers, it sends them daily email notifications about games released that day in genres there are interested in and subscribed to. This allows them to stay in the loop and gives them and opportunity to experience new games.
```
Comment (hhm970) [inline pipeline/notification/README.md:6]: Good mention of the target group being gamers for notifications!

**Review by hhm970 — DISMISSED** [review]
I'm satisfied with the updated `README`s, especially with The Why sections.

I'm going to approve this, but I think this overlaps with another pull request that is currently active, so make sure to merge that one first!

**Review by hhm970 — APPROVED** [review]
Looks all good! Extremely well-documented `README.md`s with attention to detail on the target groups
